platform/osgi/platform-osgi-condpermadmin/src/test/resources/grant.policy

```diff
@@ -0,0 +1,17 @@
+priority "grant";
```

Comment:
❓ Can we include examples of principal and signer conditions in these test policies?

Reply:
Sure.